backend/models/user.js

```diff
@@ -20,8 +20,9 @@ module.exports = (sequelize, DataTypes) => {
     restrictions: DataTypes.STRING
   }, {
     sequelize,
+		timestamps: false,
```

Comment:
Same here, maybe its github display, but it looks like your changes are space differently then exisitng ones